### PR TITLE
Extend information about Promises.

### PR DESCRIPTION
**Pull request by githesp**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Rename the folder to reflect title changes

File: guide/english/miscellaneous/the-es6-promises/index.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+Promises are a feature of ES6 to help with async programming, such as calls to a database or API.
 
-## syntax
-
-    new Promise(executor);
-    new Promise(function(resolve, reject) { ... });
+An ES6 Promise is quite similar to a normal everyday promise - an agreement to do something in future, which can be kept or broken. When you are working with async code, you generally want to wait until that async process has finished before you continue. You also want to know the outcome of that process - whether it worked (promise kept) or not (promise broken). Promises help to keep async code easy to understand.
 
-![MDN image](//discourse-user-assets.s3.amazonaws.com/original/2X/8/803d67f7e823e9e50b55debaa82bb661543d2ddc.png)
+## syntax
```
Comment (thecodingaviator) [inline guide/english/miscellaneous/the-es6-promises/index.md]:
```suggestion
### Syntax
```

File: guide/english/miscellaneous/the-es6-promises/index.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: The Es6 Promises
+title: ES6 Promises
 ---
-A new feature of ES6 to help with async programming.
+Promises are a feature of ES6 to help with async programming, such as calls to a database or API.
```
Comment (thecodingaviator) [inline guide/english/miscellaneous/the-es6-promises/index.md]:
```suggestion
## ES6 Promises
Promises are a feature of ES6 to help with async programming, such as calls to a database or API.
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**